src/Employer/Employer.Web/Orchestrators/VacancyPreviewOrchestrator.cs

```diff
@@ -1,34 +1,23 @@
-﻿using Esfa.Recruit.Employer.Web.Extensions;
-using Esfa.Recruit.Employer.Web.Services;
-using Esfa.Recruit.Employer.Web.ViewModels;
+﻿using Esfa.Recruit.Employer.Web.ViewModels;
```

Comment:
Due to the changes I'm currently making can you rollback the changes to this file and I can implement instead if appropriate.